web/src/emoji.ts

```diff
@@ -76,27 +135,28 @@ const zulip_emoji = {
     // server-side markdown, which doesn't want to render it into the
     // message content.
     emoji_url: "/static/generated/emoji/images/emoji/unicode/zulip.png",
+    still_url: undefined,
```

Comment:
Would `null` would be more correct? I think the server will always send `null` or a string, not `undefined`, which isn't a possible value in the Zulip API.